### DISCUSSION TITLE
The news notification keeps displaying

### DISCUSSION THREAD

**Discussion by QuarkMuncher** [nvim-neorg/neorg#592] [Q&A]
Every time I stat nvim, the Neorg news notification keeps showing up, even though I have read the news, how do I stop it from showing up?

**Comment by xfzv** [comment]
Same here. Running `:Neorg news new` does not make any difference, the message keeps coming back every single time I start Neovim:

![image](https://user-images.githubusercontent.com/78810647/192549558-056fe7c8-ea18-46ba-87fc-8d5fbb3bb30c.png)

**Comment by vhyrro** [comment]
Oops, i think i know what the issue is. I'll have a crack at fixing it later today :)

**Reply by vhyrro** [reply]
Okay, this isn't good. I'm looking through the code but all the logic is fine. Try deleting `stdpath("data") .. "/neorg.mpack"` perhaps? On linux this would be `~/.local/share/nvim/neorg.mpack`. Something must've borked itself in the process :shrug:

**Reply by xfzv** [reply]
Removing `~/.local/share/nvim/neorg.mpack` worked for me. The message no longer appears when starting Neovim.

Edit: nevermind, it's back after a while. :thinking:

**Reply by QuarkMuncher** [reply]
@xfzv After how long did it return? :thinking:

**Reply by vhyrro** [reply]
Okay that tells me something's wrong with the version checking after all, oops. I'll look into it again today.

**Reply by xfzv** [reply]
> @xfzv After how long did it return? thinking

A few hours.

**Comment by xfzv** [comment]
Any fix for this?

I keep getting the message after updating to 0.0.17. Manually removing `~/.local/share/nvim/neorg.mpack` doesn't make any difference, it keeps coming back.

**Reply by vhyrro** [reply]
I genuinely have zero idea why this is happening, I haven't been able to reproduce nor fix it yet :(

**Reply by vhyrro** [reply]
I will add a bunch more logging to that module to help track down the issue, give me a bit.

**Reply by vhyrro** [reply]
Alright could you update Neorg, enable trace logging and try to send over the logs?
First of all clear `~/.local/share/nvim/neorg.log`, start Neovim, close Neovim and send the content of `~/.local/share/nvim/neorg.log` here please :)

In case the content is very long grep for "cached version" and you should see a couple of messages related to versioning, copy those over instead.

Here's a snippet to enable trace logging:
```lua
require("neorg").setup({
	load = {},
	logger = {
		level = "trace",
	},
})
```

**Reply by xfzv** [reply]
> Alright could you update Neorg, enable trace logging and try to send over the logs? First of all clear `~/.local/share/nvim/neorg.log`, start Neovim, close Neovim and send the content of `~/.local/share/nvim/neorg.log` here please :)
> 
> In case the content is very long grep for "cached version" and you should see a couple of messages related to versioning, copy those over instead.
> 
> Here's a snippet to enable trace logging:
> 
> ```lua
> require("neorg").setup({
> 	load = {},
> 	logger = {
> 		level = "trace",
> 	},
> })
> ```

`~/.local/share/nvim/neorg.log`:

```console
[...]

[INFO  Sat 19 Nov 2022 08:47:08 PM CET] .../nvim/site/pack/packer/start/neorg/lua/neorg/modules.lua:213: Successfully loaded module core.norg.news
[TRACE Sat 19 Nov 2022 08:47:08 PM CET] .../start/neorg/lua/neorg/modules/core/norg/news/module.lua:35: Retrieved cached version from storage: 0.0.15
[INFO  Sat 19 Nov 2022 08:47:08 PM CET] .../nvim/site/pack/packer/start/neorg/lua/neorg/modules.lua:27: Loading module with name core.norg.qol.toc
[INFO  Sat 19 Nov 2022 08:47:08 PM CET] .../nvim/site/pack/packer/start/neorg/lua/neorg/modules.lua:130: Module core.norg.qol.toc has dependencies. Loading dependencies first...
[TRACE Sat 19 Nov 2022 08:47:08 PM CET] .../nvim/site/pack/packer/start/neorg/lua/neorg/modules.lua:134: Loading submodule core.integrations.treesitter
[TRACE Sat 19 Nov 2022 08:47:08 PM CET] .../nvim/site/pack/packer/start/neorg/lua/neorg/modules.lua:151: Module core.integrations.treesitter already loaded, skipping...
[TRACE Sat 19 Nov 2022 08:47:08 PM CET] .../nvim/site/pack/packer/start/neorg/lua/neorg/modules.lua:134: Loading submodule core.ui
[TRACE Sat 19 Nov 2022 08:47:08 PM CET] .../nvim/site/pack/packer/start/neorg/lua/neorg/modules.lua:151: Module core.ui already loaded, skipping...
[TRACE Sat 19 Nov 2022 08:47:08 PM CET] .../nvim/site/pack/packer/start/neorg/lua/neorg/modules.lua:134: Loading submodule core.keybinds
[TRACE Sat 19 Nov 2022 08:47:08 PM CET] .../nvim/site/pack/packer/start/neorg/lua/neorg/modules.lua:151: Module core.keybinds already loaded, skipping...
[TRACE Sat 19 Nov 2022 08:47:08 PM CET] .../nvim/site/pack/packer/start/neorg/lua/neorg/modules.lua:134: Loading submodule core.mode
[TRACE Sat 19 Nov 2022 08:47:08 PM CET] .../nvim/site/pack/packer/start/neorg/lua/neorg/modules.lua:151: Module core.mode already loaded, skipping...
[TRACE Sat 19 Nov 2022 08:47:08 PM CET] .../nvim/site/pack/packer/start/neorg/lua/neorg/modules.lua:134: Loading submodule core.norg.esupports.hop
[TRACE Sat 19 Nov 2022 08:47:08 PM CET] .../nvim/site/pack/packer/start/neorg/lua/neorg/modules.lua:151: Module core.norg.esupports.hop already loaded, skipping...

[...]
```

**Reply by vhyrro** [reply]
Okay yeah it's definitely lagging behind. That means something must be incorrectly overwriting the value :thinking:

**Reply by vhyrro** [reply]
Alright @xfzv try updating Neorg again and see if the issue is still there! I'm fairly sure I fixed it :flushed: 

Run `:Neorg news new`, restart Neovim and it should not show the news screen anymore.

**Reply by xfzv** [reply]
> Alright @xfzv try updating Neorg again and see if the issue is still there! I'm fairly sure I fixed it flushed
> 
> Run `:Neorg news new`, restart Neovim and it should not show the news screen anymore.

Indeed, `:Neorg news new` shows a blank window now. :v: 

While you are here :disguised_face:  - not sure if you noticed my message a couple of days ago on the concealer issues thread (https://github.com/nvim-neorg/neorg/issues/292#issuecomment-1311633262), is this feature still being worked on? just curious.

Thank you.

**Reply by vhyrro** [reply]
Ahh, yeah that's still on the agenda. I'm sure I implemented this somewhere already but I believe the code got lost in an accidental `git restore` call :/

Added to my TODO list again, it's a fairly quick thing to do so expect it sometime soon. Don't have a concrete deadline though as I'm tryharding that upgrade tool right now :p. Might do it tomorrow in fact inbetween fixing upgrader bugs.

**Reply by xfzv** [reply]
Glad to hear! it's not a big deal of course but it would definitely be useful as far as I'm concerned.

Keep up the good work
